presto-product-tests/conf/docker/singlenode-ldap/compose.sh

```diff
@@ -6,6 +6,5 @@ source "${BASH_SOURCE%/*}/../common/compose-commons.sh"
 
 docker-compose \
     -f ${BASH_SOURCE%/*}/../common/standard.yml \
-    -f ${BASH_SOURCE%/*}/../common/jdbc_db.yml \
```

Comment:
you haven't changed tests suites for this this environment, are you sure that it was not using jdbc connectors?

Reply:
Although `jdbc_db` was included it was never provisioned

https://github.com/prestodb/presto/blob/master/presto-product-tests/bin/run_on_docker.sh#L150